Project.toml

```diff
@@ -38,11 +38,11 @@ CreateParametersExt = "CLIMAParameters"
 Adapt = "3, 4"
 ArtifactWrappers = "0.2"
 CFTime = "0.1"
-CLIMAParameters = "0.8.6"
+CLIMAParameters = "0.9"
```

Comment:
could we keep the previous compat (e.g. `CLIMAParameters = "0.8.6, 0.9"`), or do these updates not have backwards compatibility? (for all the packages, not just climaparameters)

Reply:
Older versions of CLIMAParameters are compatible for now, but #481 will require v0.9 or greater for the new parameters. I could change the compat again in in that PR, I'm not sure what's preferable.

The other packages all require CP v0.9. We could keep their older versions in the compat, but I think the restrictions with climaparameters will force the package manager to grab the newer versions.

Reply:
Okay! Changing it to just the latest versions in this PR is fine